### PR TITLE
x11spice: init at 2019-08-20

### PR DESCRIPTION
**Pull request by rnhmjoj**
###### Motivation for this change

SPICE is a protocol generally used for virtual machines but it's also useful as a remote desktop: it's much faster than regular VNC.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change (new package)
- [ ] Tested execution of all binary files
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/X11/x11spice/default.nix [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+stdenv.mkDerivation rec {
+  pname   = "x11spice";
+  version = "2019-08-20";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.freedesktop.org";
+    owner  = "spice";
+    repo   = "x11spice";
+    rev    = "51d2a8ba3813469264959bb3ba2fc6fe08097be6";
+    sha256 = "0va5ix14vnqch59gq8wvrhw6q0w0n27sy70xx5kvfj2cl0h1xpg8";
```
Comment (Lassulus) [inline pkgs/tools/X11/x11spice/default.nix]: I'm personally not a big fan of this indentation style, it either creates unnecessary diff noise when adding a new attribute which is longer. If we now have something to automatically format things like this, I would be happy for a heads up. If you really prefer this style of indent I would also let it pass

Comment (rnhmjoj) [reply pkgs/tools/X11/x11spice/default.nix]: No, it's nothing automatic: I have been formatting like this since I started contributing in 2015, which is a habit I got from writing haskell.
Since you are the second reviewer that recently disliked it I'm just going to drop it to. I'm just surprised that no one ever mentioned this before (I did around 300 PRs) if it's not generally accepted.

Comment (rnhmjoj) [reply pkgs/tools/X11/x11spice/default.nix]: Done.

**Review by Lassulus — APPROVED** [review]
tested with nix-review, started the server, connectet with a client

**Comment by rnhmjoj** [comment]
Thank you.
